yarn.lock

```diff
@@ -15038,9 +15038,9 @@ preact@10.15.1:
   resolved "https://registry.yarnpkg.com/preact/-/preact-10.15.1.tgz#a1de60c9fc0c79a522d969c65dcaddc5d994eede"
   integrity sha512-qs2ansoQEwzNiV5eAcRT1p1EC/dmEzaATVDJNiB3g2sRDWdA7b7MurXdJjB2+/WQktGWZwxvDrnuRFbWuIr64g==
 
-"prebid.js@github:guardian/prebid.js":
+prebid.js@guardian/prebid.js:
```

Comment:
I have absolutely no idea. Though this change does bring it into line with other entries:
<img width="783" alt="Screenshot 2023-08-16 at 16 50 13" src="https://github.com/guardian/dotcom-rendering/assets/5357530/077fd068-d268-4fa5-8b52-702ad97d4cda">

I thought it might have had quotes because of the `.js` - but other packages with similar also lack quotes:
<img width="786" alt="Screenshot 2023-08-16 at 16 52 42" src="https://github.com/guardian/dotcom-rendering/assets/5357530/cdba2360-3f94-4ff8-a670-40880086e668">